ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,6 @@
+Note: if you are reporting a wrong signature of a function or a class in
+the standard library, then typeshed tracker is better suited for this report.
```

Comment:
"... then _the_ typeshed tracker ..." Also I'd add a link.

Reply:
Note that the template is shown in plain text, so I don't use any Markdown for the link.